.github/workflows/reusable-run-danger.yml

```diff
@@ -0,0 +1,34 @@
+on:
+  workflow_call:
+    inputs:
+      remove-previous-comments:
+        description: 'Configures Danger to always remove previous comments and add a new one instead of editing the same comment.'
+        default: true
+        type: boolean
+        required: false
+    secrets:
+      github-token:
+        required: true
+
+concurrency:
+  group: danger-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  dangermattic:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "📥 Checkout Repo"
+        uses: actions/checkout@v4
+      - name: "💎 Ruby Setup"
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+      - name: "☢️ Danger PR Check"
+        env:
+          REMOVE_PREVIOUS_COMMENTS: ${{ inputs.remove-previous-comments }}
+          DANGER_GITHUB_API_TOKEN: ${{ secrets.github-token }}
+        run: |
+          echo "--- 🏃 Running Danger: PR Check"
+          bundle exec danger --fail-on-errors=true ${REMOVE_PREVIOUS_COMMENTS:+--remove-previous-comments} --danger_id=pr-check
```

Comment:
Today I re-learned about `$(foo:-val}` bash syntax (always glad to have https://devhints.io/bash bookmarked as quick ref for those!) 👍

Reply:
Oh I didn't know about this site, nice one! 👍